lib/PromiseQueue.ts

```diff
@@ -70,7 +70,7 @@ export class PromiseQueue {
     // unnecessarily complex ES3 compatible code.
     task()
       .then(resolve, reject)
-      .catch(() => {
+      .finally(() => {
```

Comment:
To retain backwards compability, this can also be written as:
```suggestion
      // Mimic .finally() by chaining .catch().then()
      .catch(() => void 0)
      .then(() => {
```